pkg/kubelet/cm/container_manager.go

```diff
@@ -132,6 +133,12 @@ type ContainerManager interface {
 	// might need to unprepare resources.
 	PodMightNeedToUnprepareResources(UID types.UID) bool
 
+	// UpdateAllocatedResourcesStatus updates the status of allocated resources for the pod.
+	UpdateAllocatedResourcesStatus(pod *v1.Pod, status *v1.PodStatus)
+
+	// Updates returns a channel that receives an Update when the device changed it's status.
+	Updates() <-chan resourceupdates.Update
```

Comment:
Similarly, I think this could be named ResourceUpdates

Reply:
I like the Update() more. Like Prober manager and others - they all have `Updates()`. There is nothing else semantically can refer to updates from the container manager. At least I am having hard time imagining.